tests/ui/conftest.py

```diff
@@ -95,29 +297,29 @@ def user(create_superuser, my_base_url, fxa_account, jwt_token):
 @pytest.fixture(scope='function')
 def live_server(request, transactional_db):
```

Comment:
Perhaps I'm not understanding the issue? We shouldn't need a clean database for every test, but every test should be able to generate unique data that's required for the test to run. This is why returning an object describing the generated content would be useful.